config/routes.rb

```diff
@@ -29,6 +29,9 @@
     resources :user_roles, only: %i[edit update]
     resources :proposals, only: %i[index]
     resources :calendars, only: %i[index]
+
+    get 'forum', to: 'forums#index'
```

Comment:
Acho que se usar `resources :forum, only: %i[index], to: 'forums#index'` vai dar no mesmo resultado, e vai estar sendo usado o `resources`, que acredito ser mais recomendado.

Reply:
A gente usou assim só para que a palavra `/forum` na url ficasse no singular. Pensamos no `resource`, mas ele não aceita a action index, já que é algo do rails que interpreta que não exista nenhum tipo de listagem.